local.py

```diff
@@ -8,8 +8,8 @@
 from absl import app
 from absl import flags
 
-MODEL = "UPR_2percent_ps0"
-INSTANCE = "1573031002"
+DEFAULT_MODEL = "UPR_2percent_ps0"
```

Comment:
why not load flask_config here? I know it's not technically 'local', but sharing config seems better.

Reply:
I deleted the flask_config.py altogether now. I could easily also read in the model config for the out information...